src/connector.js

```diff
@@ -418,7 +431,12 @@ function initEngine() {
 
 						// filter user sensitive content
 						requestContent.enableQuerySyntax = params.isAdvancedSearch;
-						requestContent.mlParameters = { "filters": { "c_context_searchpageurl": params.originLevel3 } };
+						requestContent.mlParameters = { 
+							"filters": { 
+								"c_context_searchpageurl": params.originLevel3, 
+								"c_context_searchpagerelativeurl": params.originLevel3RelativeUrl 
```

Comment:
Based on proposed approach, it is not a configurable parameter anymore, thus:
```suggestion
								"c_context_searchpagerelativeurl": originLevel3Relative
```